RHDHBUGS-261: Operator: Add documentation for default config (#1384)
redhat-developer/red-hat-developers-documentation-rhdh: modules/configuring-external-databases/con-automated-operator-features.adoc

:_mod-docs-content-type: CONCEPT

[id="con-automated-operator-features_{context}"]
= Automated Operator features

> The Operator automates several key processes to ensure your {product-custom-resource-type} application is configured correctly and consistently.

Reviewer: ```suggestion
You can use the Operator to automate several key processes to effectively configure your {product-custom-resource-type} application.
```
(Trying to make it active voice, and user-centric language. Is this technically accurate, though?)

Reviewer: Please ignore if this does not apply.
